### PR TITLE
Add VAST endcard spec

### PR DESCRIPTION
**Pull request by arimura**
Add VAST endcard spec

### CODE REVIEW

**Review by suzuken — APPROVED** [review]
LGTM

**Review by yowcow — CHANGES_REQUESTED** [review]
Let's have VAST spec defined outside of RTB spec

File: vast-v1.en.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# fluct VAST Specification ver 2.2
```
Comment (yowcow) [inline vast-v1.en.md]: `s/2.2/1.0/` :bulb:

File: vast-v1.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+* [0. 変更履歴](#0-変更履歴)
+* [1. VASTのエンドカード仕様](#0-VASTのエンドカード仕様)
+
+## 0. 変更履歴
+
+[CHANGELOG-VAST.md](CHANGELOG-VAST.md)
+
+## 1. VASTのエンドカード仕様
+
+動画リワード等で利用されるエンドカードは、fluctに返却されるレスポンスが含むVASTの `<Inline>` にある `<CompanionAds>` として配置して下さい。
```
Comment (yowcow) [inline vast-v1.md]: `s/Inline/InLine/` :bulb:

**Review by yowcow — APPROVED** [review]
LGTM :dash:

**Pull request merged**